main/resource-access.bs

```diff
@@ -67,13 +69,35 @@ if it needs to access resources requiring authentication (see [[#webid]]).
 When it receives a response with a `403` or `404` status code,
 it MAY repeat the request with different credentials.
 
+A Solid client MUST use the `Content-Type` request header in `PUT`, `POST` and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/70#issuecomment-547924171)]
 
-## Linked Data Platform ## {#ldp}
+## Uniform Resource Identifier ## {#uri}
 
-Issue: Write Linked Data Platform section.
+### Required shared slash semantics ### {#uri-path-component}
 
-Draft:
-A Solid data pod MUST conform to the LDP specification [[!LDP]].
+Server and client share the slash semantics in the URI path component representing hierarchy [[!RFC3986]]. [[Source](https://github.com/solid/specification/issues/35#issuecomment-547949014)]
+
+Server MUST NOT create a URI differing only in the trailing slash with that of an existing URI. [[Source](https://github.com/solid/specification/issues/107#issuecomment-567482817)]
+
+Server MAY redirect a request with 301 from a non-existing URI to an existing URI with a matching path component ending with the same segment term. Behaviour pertaining to authorization MUST proceed this optional redirect. [[Source](https://github.com/solid/specification/issues/107#issuecomment-567454889)]
+
+## Linked Data ## {#linked-data}
+
+### Containment ### {#resource-containment}
+
+Solid has the notion of containers to represent a collection of linked resources to help with resource discovery and lifecycle management.
+
+There is a 1-1 correspondence between containment triples and relative reference within the path name hierarchy. [[Source](https://github.com/solid/specification/issues/98#issuecomment-547506617)]. It follows that all resources are discoverable from a container and that it is not possible to create orphan resources. [[Source](https://github.com/solid/specification/issues/97#issuecomment-547459396)]
+
+The representation and behaviour of containers in Solid is corresponds to LDP Basic Container and MUST be supported. [[Source](https://github.com/solid/specification/issues/47#issuecomment-561675764)]
```

Comment:
Does this mean we cannot have other container types?

Reply:
No. The statement only mentions what's required. Optionals or recommendations against anything is not mentioned - normally they're allowed but would be out of spec behaviour. The linked source and https://github.com/solid/specification/issues/47#issuecomment-587421049 suggests to leave the descriptions for the other cases (if at all) for later.

Reply:
OK; just noticed typo _is corresponds_.

Reply:
Thanks. Caught that earlier https://github.com/solid/specification/pull/157/commits/53fbc185e1a7e6d1d151870c94973ed9e8d60496